src/classifier_facevectors.js

```diff
@@ -77,7 +77,7 @@ async function main() {
 	}
 }
 
-tf.setBackend(process.env.RECOGNIZE_PUREJS === 'true' ? 'wasm' : 'tensorflow')
```

Comment:
Only checking the env var is not enough, because if loading tf-node fails we want to switch to wasm dynamically